iio-client.h

```diff
@@ -32,26 +32,63 @@
 #include "custom-libiio-client/iio.h"
 
 #define MAX_SENSOR 9
+#define MAX_CHANNEL 3
+#define kDefaultMinDelayUs (10 * 1000)
+#define kDefaultMaxDelayUs (200 * 1000)
 
 struct idMap {
     const char *name;
     int id;
     int type;
 };
 
+struct event_info {
+    const char* name;
+    int handle;
+    sensors_event_t* event;
+    int data_count;
+    float scale;
+};
+
+struct iioclient_device {
+    const char *name;
+    const struct iio_device *dev;
+    int type;
+    double scale;
+    int raw_channel_count;
+    struct iio_channel *channel_raw[10];
+	struct iio_channel *channel_frequency;;
```

Comment:
remove 1 extra ;
coding style not in line with rest of the files
Indentation issues. please check the whole file for indentation issues